_config.dev.yml

```diff
@@ -0,0 +1 @@
+url: http://localhost:4000
```

Comment:
Can you add a comment explaining? Possibly a link to the 3.3 jekyll image request issue